quipucords/scanner/vcenter/inspect.py

```diff
@@ -151,10 +150,13 @@ def parse_host_props(self, props, cluster_dict):
         facts = {}
         for prop in props:
             if prop.name == 'parent':
-                facts['host.cluster'] = \
-                    cluster_dict[str(prop.val)]['cluster.name']
-                facts['host.datacenter'] = \
-                    cluster_dict[str(prop.val)]['cluster.datacenter']
+                c_src = cluster_dict.get(str(prop.val))
+                if c_src is not None:
+                    facts['host.cluster'] = c_src.get('cluster.name')
```

Comment:
Since you using get, you could leverage it by changing from line 153 to 159 to something like:

```python
c_src = cluster_dict.get(str(prop.val), {})
facts['host.cluster'] = c_src.get('cluster.name')
facts['host.datacenter'] = c_src.get('cluster.datacenter')
```